geode-core/src/main/java/org/apache/geode/management/internal/api/ClusterManagementService.java

```diff
@@ -19,15 +19,21 @@
 import org.apache.geode.cache.configuration.CacheElement;
 
 /**
- * Defines the behavior for a management service object, which is responsible for applying and
- * persisting
- * cache configuaration changes on locators and/or servers.
+ * this is responsible for applying and persisting cache configuration changes on locators
+ * and/or servers.
  */
 @Experimental
 public interface ClusterManagementService {
-  APIResult createCacheElement(CacheElement config);
+  /**
+   * This method will try to create the element on all the applicable members in the cluster and
+   * persist the configuration in the cluster configuration if persistence is enabled.
+   *
+   * @param config this holds the configuration attributes of the element you are trying to create
+   *        on the cluster
+   */
+  ClusterManagementResult createCacheElement(CacheElement config);
 
-  APIResult deleteCacheElement(CacheElement config);
+  ClusterManagementResult deleteCacheElement(CacheElement config);
 
-  APIResult updateCacheElement(CacheElement config);
+  ClusterManagementResult updateCacheElement(CacheElement config);
```

Comment:
It seems a bit redundant having 'CacheElement' in the name of these methods. It's also not entirely accurate seeing that the methods will result in both persistence and realization.

Reply:
sounds good.